contrib/verifybinaries/verify.sh

```diff
@@ -25,7 +25,7 @@ TMPFILE="hashes.tmp"
 SIGNATUREFILENAME="SHA256SUMS.asc"
 RCSUBDIR="test"
 HOST1="https://github.com/dashpay/dash/releases/download/v"
-HOST2="https://pasta.keybase.pub/Dash-Core-Releases/"
+HOST2="https://dashcore-binaries.thepasta.org/file/dashcore-binaries/"
```

Comment:
why not dash.org then?

Reply:
Dash.org doesn't host any binaries at this point at least

Reply:
we can setup releasing process to upload this binaries before official release but not somedays later as now

Reply:
Huh? dash.org/downloads simply links to the GitHub artifacts